### PR TITLE
Feature/install button implementation

### PR DESCRIPTION
**Pull request by edward-mtsh**
Added install button implementation.

### CODE REVIEW

File: Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift [hunk 1]
```diff
@@ -10,8 +10,18 @@ class MainDetailView: UIView {
     @IBOutlet private weak var productName: UILabel!
     @IBOutlet private weak var clientName: UILabel!
     @IBOutlet private weak var shortDescription: UILabel!
+    @IBOutlet private weak var installButton: UIButton!
     //MARK: LifeCycle
     
+    private let dependencyContainer = DependencyContainer.container()
+    private var sharedApplication: SharedApplicationDelegate?
```
Comment (pjwelcome) [inline Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift]: why does this need to be a global variable. you only use it in one place when you resolve for this delegate on the container, the container doesnt need to be a global either.

File: Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift [hunk 2]
```diff
@@ -54,5 +66,13 @@ class MainDetailView: UIView {
             }
         }
     }
+    
+    @IBAction func onInstallButtonTapped(_ sender: Any) {
+        guard let iOSPackageName = showcaseApp?.iosPackageName, let url = URL(string: iOSPackageName) else {
```
Comment (pjwelcome) [inline Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift]: All this should be in the presenter and should be tested. you should ask the presenter to open the Url

**Review by pjwelcome — CHANGES_REQUESTED** [review]

File: Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift (hunk 1 above)
Comment (edward-mtsh) [reply Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift]: Im using it to resolve the FIRStoring and SharedApplicationDelegate

File: Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift (hunk 2 above)
Comment (edward-mtsh) [reply Showcase-iOS/Dashboard/DetailView/MainDetailSubView/MainDetailView.swift]: Ok i will update the PR with tests and move code to the presenter.

**Review by pjwelcome — APPROVED** [review]

**Pull request merged**